web/src/components/core/FileViewer.jsx

```diff
@@ -62,16 +62,16 @@ export default function FileViewer({ file, title, onCloseCallback }) {
       variant="large"
       className="tallest"
     >
-      { state === "loading" && <LoadingEnvironment text="Reading file..." /> }
-      { (content === null || error) &&
+      {state === "loading" && <LoadingEnvironment text="Reading file..." />}
+      {(content === null || error) &&
         <Alert
           isInline
           isPlain
           variant="warning"
           title="Cannot read the file"
         >
           {error}
-        </Alert> }
+        </Alert>}
```

Comment:
I guess this change is just for making ESLint happy, as it's partially done in https://github.com/yast/d-installer/pull/418#pullrequestreview-1289770888